bears/general/CPDBear.py

```diff
@@ -127,4 +128,5 @@ def run(self, language: str, minimum_tokens: int=20,
                         'that you have more code than you need. Consider'
                         ' refactor your code to remove one of the'
                         ' occurrences. For more information go here:'
-                        'http://tinyurl.com/coala-clone'))
+                        'http://tinyurl.com/coala-clone'),
```

Comment:
Does tinyurl support previewing where you go similar to bit.ly? if not, I'd prefer using bit.ly or just the long url

Reply:
We shouldnt be using external shorteners.  We have our own shortener at coala.io :/